src/sys.zig

```diff
@@ -540,6 +540,10 @@ pub fn normalizePathWindows(
         } };
     };
 
+    if (path.len >= 2 and path[0] >= 'A' and path[0] <= 'Z' and path[1] == ':') {
+        path = path[2..];
```

Comment:
Don't we have a function for matching the disk drive pattern?

Reply:
will change to `bun.path.isDriveLetter(path[0])` once the tests pass